apps/confidential/confidential_test.go

```diff
@@ -65,19 +66,19 @@ func TestCertFromPEM(t *testing.T) {
 
 const (
 	authorityFmt      = "https://%s/%s"
-	fakeAuthority     = "https://fake_authority/fake"
+	fakeAuthority     = "https://fake_authority/fake_tenant"
 	fakeClientID      = "fake_client_id"
 	fakeSecret        = "fake_secret"
-	fakeTokenEndpoint = "https://fake_authority/fake/token"
+	fakeTokenEndpoint = "https://fake_authority/fake_tenant/token"
 	localhost         = "http://localhost"
 	refresh           = "fake_refresh"
 	token             = "fake_token"
 )
 
 var tokenScope = []string{"the_scope"}
 
-func fakeClient(tk accesstokens.TokenResponse, credential Credential, options ...Option) (Client, error) {
-	client, err := New(fakeAuthority, fakeClientID, credential, options...)
+func fakeClient(tk accesstokens.TokenResponse, credential Credential, fAuthority string, options ...Option) (Client, error) {
```

Comment:
Pls undo this name shortnening as it is harder to read the code.

Reply:
expanded